cdm_cut.sql

```diff
@@ -0,0 +1,166 @@
+/**
+ * CDM cut for DS-DETERMINED cohort
+ */
+DROP TABLE condition;
+
+CREATE TABLE condition AS
+SELECT
+    c.*
+FROM
+    cdm60_deid_dataset."condition" c
+    JOIN pcornet_trial pt ON pt.patid = c.patid;
```

Comment:
Typically pcornet trial table has patients from multiple trails, You may want to pick only ds trail patients

select distinct trialid  from cdm60_limited_dataset.pcornet_trial pt ;
/*
ADPT
PREVENTABLE
PS_CMSLINKAGE_PILOT
*/